app/models/document.rb

```diff
@@ -142,7 +143,7 @@ class NotArchiveableError < StandardError; end
   validate :numbered
   validate :created_date_is_in_the_past
 
-  default_scope -> { no_owner.or(not_excluded_owners) }
+  default_scope -> { no_owner.or(not_excluded_owners).not_for_fee_exemption }
```

Comment:
As discussed, we should remember to review how we handle different sets of documents in the near future, especially if we're going to be adding more mutually-exclusive sets.